ckanext/switzerland/templates/package/read_base.html

```diff
@@ -0,0 +1,8 @@
+{% ckan_extends %}
+
+{% block content_primary_nav %}
+  {{ h.build_nav_icon('dataset_read', _('Dataset'), id=pkg.name) }}
+  {{ h.build_nav_icon('dataset_groups', _('Categories'), id=pkg.name, ckan_icon='folder') }}
```

Comment:
This tab was being displayed as "Groups" instead of "Categories", so overwrote the template to use the right wording.